lib/rubocop/cop/style/documentation.rb

```diff
@@ -47,18 +47,25 @@ def check(ast, ast_with_comments)
           end
         end
 
+        def module_definition?(node)
+          # 'Const = Class.new' or 'Const = Module.new' are module definitions
+          node.class_type? ||
+            node.module_type? ||
+            (node.casgn_type? &&
+              (rhs = node.children[2]).send_type? &&
```

Comment:
Yeah, I'm well aware of this, but as you said - others might not be. :-) That's not a major point, though.